mtt.py

```diff
@@ -45,23 +41,22 @@ def get_laplacian(adj_matrix):
         row[i] = degree
     return L
 
-                
-def MTT(graph):
+# return in natural log scale by default
```

Comment:
Did you mean default to be without log scale?

Reply:
Thanks, fixed the comment!